experimental/Origami/src/deck_group.jl

```diff
@@ -0,0 +1,54 @@
+function in_deck_group(o::Origami, sigma::PermGroupElem)
+    h = horizontal_perm(o)
+    v = vertical_perm(o)
+    for i in 1:degree(o)
+        if (i^sigma)^h != (i^h)^sigma || (i^sigma)^v != (i^v)^sigma
+            return false
+        end
+    end
+    return true
```

Comment:
Isn't this the same as
```suggestion
    return sigma*h == h*sigma && sigma*v == v*sigma
```
or, even better,
```suggestion
    return h == h^sigma && v == v^sigma
```
(at least assuming that `sigma` is also in `Sym(degreeo))`)